### DISCUSSION TITLE
Trace Profiling has no analysis result and is loading all the time

### DISCUSSION THREAD

**Discussion by BertOnline** [apache/skywalking#9906] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/17882970/199908727-59ba1189-519d-46fd-8d2d-c4d9e6d7a916.png)

Trace Profiling has no analysis result and is loading all the time. When Exclude Children is selected. What caused this?

**Comment by wu-sheng** [comment]
@mrproliu

**Comment by mrproliu** [comment]
If you want to use exclude children mode to analyze, please select one slow span first. Otherwise, the system cannot know which span you want to analysis.

**Reply by wu-sheng** [reply]
@Fine0830 Could you check whether 9.3.0 UI fixed this ?

**Reply by Fine0830** [reply]
It's a UI bug. I will fix it.

**Comment by wu-sheng** [comment, marked as answer]
UI fix, https://github.com/apache/skywalking-booster-ui/pull/179

**Reply by BertOnline** [reply]
@wu-sheng 
![image](https://user-images.githubusercontent.com/17882970/199982004-471567d8-9436-46e9-afad-0ebc039efeff.png)
The top slow button is invalid. I wonder if it is a bug?

**Reply by wu-sheng** [reply]
What do you meam invalid? The style is like this. We don't have a very specific style for it.

**Reply by BertOnline** [reply]
@wu-sheng Thanks for your reply. It's my mistake. Its function is to highlight time-consuming information.

**Reply by wu-sheng** [reply]
Yes, it should highlight slow methods. Do you mean it is not working?

**Reply by BertOnline** [reply]
It works normally. I thought it would rearrange slow methods and highlight topN. I was wrong and my idea was unreasonable. Thank your.
